### ISSUE TITLE
Metadata does not work with Text logging

### ISSUE THREAD

**Issue by jaygorrell** [peopledata/fluentd#6]
I believe the expectation of `KUBERNETES_META` should remain the same regardless of the logging format used. Here is how I'm seeing things right now:

`LOG_FORMAT` | `KUBERNETES_META` | Results
--- | --- | ---
text | false | Text Logs
json | false | Json Logs
merge_json_log | false | Json Logs (merged)
text | true | Text Logs
json | true | Json Logs with Metadata
merge_json_log | true | Json Logs (merged) with Metadata

I hope this shows what I mean. I would expect key=value formatted metadata when using text logs, even if the key needs to use dot.notation to show structured keys.

**Comment by Stono** [comment]
That might get messy though, the text logs were to support things like, NGINX line files which sumo can parse automatically.  If you start appending on arbitrary text you'll impact that parsing?

@stevezau  ?

**Comment by jaygorrell** [comment]
Yeah, I understand. As a minimum, documenting that `KUBERNETES_META` is only supported for the json log formats may work. I just found these options a little confusing to figure out what did what.

I could even see an argument for a `_with_meta` variation of the two json logs for an alternative route. Something that explicitly shows it doesn't go with the text option.
